interop-cats/jvm/src/main/scala/scalaz/zio/interop/catszmanagedjvm.scala

```diff
@@ -0,0 +1,102 @@
+package scalaz.zio.interop
+
+import cats.effect.{ Bracket, LiftIO, Resource }
+import cats.{ ~>, effect, Bifunctor, Defer, Functor, Monad, MonadError, Monoid, Semigroup, SemigroupK }
+import scalaz.zio.{ Reservation, ZIO, ZManaged }
+
+trait CatsZManagedSyntax {
+
+  implicit class ResourceSyntax[F[_], A](resource: Resource[F, A]) {
+    def toManaged(
+      implicit nat: F ~> ZIO[Any, Throwable, ?],
```

Comment:
If you have `ConcurrentEffect` or whatever, you can go from `F[_]` to `ZIO` already, so I'd rely on that, I think, and delete the implicit natural transformation.

Reply:
With ConcurrentEffect I seem to only be able to go to cats IO unless I'm missing a conversion method somewhere.

Reply:
With `liftIO`, you can go from `IO` to any `F[_]`, including `ZIO`.

Reply:
You are right. I'll change this

Reply:
This would limit this to Resource[cats.effect.IO, A] though. I assume this is fine?